src/routes/room.routes.ts

```diff
@@ -14,6 +14,20 @@ router.get('/:id', (req, res) => {
     });
 });
 
+router.patch('/:id', (req, res) => {
+    const { title, body } = req.body;
+    if (!title) return sendError(res, "Title can't be empty");
+    let id: number = +req.params.id;
```

Comment:
```suggestion
    let id = +req.params.id;
```
Type deduction is implicit here.